src/System.IO.Compression.Brotli/System/IO/Compression/Brotli/Brotli.cs

```diff
@@ -36,10 +42,14 @@ private static TransformationStatus GetTransformationStatusFromBrotliDecoderResu
             if (result == BrotliDecoderResult.NeedsMoreInput) return TransformationStatus.NeedMoreSourceData;
             return TransformationStatus.InvalidData;
         }
+        public static TransformationStatus Compress(ReadOnlySpan<byte> source, Span<byte> destination, out int bytesConsumed, out int bytesWritten, CompressionLevel quality = (CompressionLevel)DefaultQuality, int windowSize = DefaultWindowSize, BrotliEncoderMode encMode = BrotliEncoderMode.Generic)
```

Comment:
Because DefaultQuality is Maximum possible quality. I think we need more levels in future. Because (It is a huge difference between 11 to 10 to 5 to 2 to 1 to 0)